compliance_users/scripts/projects.py

```diff
@@ -39,7 +39,7 @@ def read_project_data(data_path):
     issuance_df["opr_id"] = issuance_df["opr_id"].astype(str).str.strip()
 
     issuance_df["project_type"] = issuance_df["project_type"].map(project_types)
-    issuance_df["arb_id"] = issuance_df["arb_id"].str.split("-").apply(lambda x: x[0])
+    issuance_df["arb_id"] = issuance_df["arb_id"].str.split("-", n=1).str[0]
```

Comment:
i think you could also do `issuance_df["arb_id"].str.split("-").list[0]`? 

reads a little cleaner. i was thinking that indexing a string might just return the first item of the string?

Reply:
you get a series back, rather than a string from str.split, so i think you don't run the risk of getting back just the first item of a string. but agree this looks clearner! i will drop this in and see if it works.